### PR TITLE
[All] Added ChartCenter Helm repository in README

### PR DESCRIPTION
**Pull request by chukka**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
[All] Added ChartCenter Helm repository in README

### CODE REVIEW

File: stable/mission-control/README.md [hunk 1]
```diff
@@ -473,7 +477,7 @@ The following table lists the configurable parameters of the mission-control cha
 | `insightScheduler.loggersResources.limits.cpu`       | Insight Scheduler loggers cpu limit              |                              |
 | `insightExecutor.name`                       | Insight Executor name                           | `insight-scheduler`                   |
 | `insightExecutor.image`                      | Container image                                 | `docker.jfrog.io/jfrog/insight-executor`   |
-| `insightExecutor.version`                    | Container image tag                             | `.Chart.AppVersion`                   |
+| `insightExecutor.image.ersion`               | Container image tag                             | `.Chart.AppVersion`                   |
```
Comment (danielezer) [inline stable/mission-control/README.md:480]: typo

File: stable/xray/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # JFrog Xray Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [2.3.0] - Jul 13, 2020
```
Comment (danielezer) [inline stable/xray/CHANGELOG.md:4]: Why does this justify a minor version bump? I think this should be 2.2.2

File: stable/xray/README.md [hunk 3]
```diff
@@ -42,8 +45,8 @@ helm status xray
 ## Upgrade
 To upgrade an existing Xray, you still use **helm**
 ```bash
-# Update existing deployed version to 2.1.2
-helm upgrade --set common.xrayVersion=2.1.2 jfrog/xray
+# Update existing deployed version to 2.14.0
```
Comment (danielezer) [inline stable/xray/README.md:48]: 2.14.0 ?

File: stable/mission-control/Chart.yaml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: mission-control
 description: A Helm chart for JFrog Mission Control
-version: 2.0.2
+version: 2.1.0
```
Comment (danielezer) [inline stable/mission-control/Chart.yaml:4]: Why does this justify a minor version bump? I think this should be 2.0.3

Comment (chukka) [reply stable/mission-control/Chart.yaml:4]: Done

File: stable/xray/README.md (hunk 3 above)
Comment (chukka) [reply stable/xray/README.md:48]: I updated to latest preunified version

File: stable/xray/CHANGELOG.md (hunk 2 above)
Comment (chukka) [reply stable/xray/CHANGELOG.md]: Done

File: stable/mission-control/README.md (hunk 1 above)
Comment (chukka) [reply stable/mission-control/README.md]: Done

**Pull request merged**